evergreen/config.yml

```diff
@@ -977,18 +989,17 @@ tasks:
       test_label: objstore-local
       test_executable_name: "realm-object-store-tests"
       verbose_test_output: true
+      disable_tests_against_baas: true
   - func: "check branch state"
 
 # These are baas object store tests that run against baas running on a remote host
 - name: baas-integration-tests
   tags: [ "test_suite", "for_pull_requests", "requires_baas" ]
```

Comment:
It is a bit confusing to have both the `disable_tests_against_baas` var and `requires_baas`. Can we have only one of these for clarity?

Reply:
Unfortunately no, `requires_baas` is a tag rather than an expansion which tells evergreen which tasks to run and `disable_tests_against_baas` is an expansion used in commands to tell them how to compile the object-store tests.